src/tpm2.c

```diff
@@ -4686,7 +4442,7 @@ TPM_RC TPM2_NV_Certify(NV_Certify_In* in, NV_Certify_Out* out)
     TPM2_CTX* ctx = TPM2_GetActiveCtx();
 
     if (ctx == NULL || in == NULL || out == NULL || ctx->authCmd == NULL)
-        return TPM_RC_BAD_ARG;
+        return BAD_FUNC_ARG;
```

Comment:
Why the change from separate error codes for wolfTPM to dependent on wolfSSL error codes?

Reply:
The TPM_RC codes are from the TCG specification and adding non-standard codes for those isn't ideal. Since we have this strong dependency on wolfSSL I thought we'd just use wolf error codes. In fact the wolfTPM2_GetRCString function calls into wc_GetErrorString for negative values.